client/settings/payment-settings/general-settings-section.js

```diff
@@ -17,87 +17,63 @@ import styled from '@emotion/styled';
  */
 import CardBody from '../card-body';
 
+const StyledCard = styled( Card )`
+	margin-bottom: 12px;
+`;
+
 const GeneralSettingsSection = () => {
 	const [ enableStripe, setEnableStripe ] = useState( false );
 	const [ enableTestMode, setEnableTestMode ] = useState( false );
 
-	const toggleEnableStripe = () => setEnableStripe( ! enableStripe );
-	const toggleEnableTestMode = () => setEnableTestMode( ! enableTestMode );
-
-	const CheckboxDescription = styled.span`
-		font-style: italic;
-		display: block;
-	`;
-
-	const GeneralSettingsOptionWrapper = styled.div`
-		&:first-of-type {
-			margin-top: 26px;
-		}
-		&:not( :last-child ) {
-			margin-bottom: 26px;
-		}
-		&:last-child {
-			margin-bottom: 24px;
-		}
-	`;
-
```

Comment:
I understand that the intent of this PR is to adjust the margin to match the other component, but these are added to match the Figma's design. 

In some of our PR, i noticed we try to match figma 100%, in some,  we don't. Instead of going back and forth on these tweaks, how should we decide when to match the design and when to not? Is it save to ignore all margin/paddings as long as we use WP components? (personally +1 to fall back to the wp component defaults)

Reply:
Yeah, it's a bit of a mess 🤦 I'd say to fallback on what we did on WCPay - in this case there is an understanding that the help text on the components in Figma isn't correctly synced to match the help text in Storybook (ie.e.: it's an issue within the Figma components - the help text _used_ to be italic in WP components).

Whenever we have doubts we can always ask UX, trying to push for the "standard" WP components' behavior so that we can reduce the maintenance burden.